### DISCUSSION TITLE
Async/Await Transpilation

### DISCUSSION THREAD

**Discussion by james0r** [WeAreAthlon/frontend-webpack-boilerplate#42] [Q&A] [answered]
The starter says supports ES7 transpilation but i'm getting a regeneratorRuntime error when using async/await.

is the @babel/Polyfill required?

Thanks!

**Comment by pnikolov** [comment, marked as answer]
@james0r 

In order to add support for `async/await` in your built code you need to modify the `.babelrc` file and include the es modules, here is the updated `.babelrc` file:

```js
{
  "presets": [
    [
      "@babel/preset-env",
      {
        "targets": {
          "esmodules": true
        }
      }
    ]
  ]
}
```

Having this the output code will correctly handle async/await in the targeted browsers.
